include/ensmallen_bits/problems/ackley_function.hpp

```diff
@@ -60,8 +60,16 @@ class AckleyFunction
   size_t NumFunctions() const { return 1; }
 
   //! Get the starting point.
+  //! This was previously -5.0, 5.0.
   template<typename MatType = arma::mat>
-  MatType GetInitialPoint() const { return MatType("-5.0; 5.0"); }
+  MatType GetInitialPoint() const { return MatType("0.02; 0.02"); }
```

Comment:
I can see you changed some of the Initial points, to be closer to the final point, is that for performance reasons?

Reply:
Yeah---they did not always seem to converge with the given initial points. So I took the starting points that were actually being used in `adam_test.cpp` and used those for `GetInitialPoint()`. I wrote the original values in a comment in case we ever want to change back.

Reply:
Not sure but I guess the comment can be confusing for users, as they don't really have a reference where this comes from, we could just browse the commit history to check what the old value was?

Reply:
Fair point---I removed those comments in 0aeab3c. :+1: